R/annotation-custom.r

```diff
@@ -44,32 +44,41 @@ NULL
 #' base +
 #'   annotation_custom(grob = g, xmin = 1, xmax = 10, ymin = 8, ymax = 10)
 annotation_custom <- function (grob, xmin = -Inf, xmax = Inf, ymin = -Inf, ymax = Inf) {
-  GeomCustomAnn$new(geom_params = list(grob = grob, xmin = xmin,
-    xmax = xmax, ymin = ymin, ymax = ymax), stat = "identity",
-    position = "identity", data = NULL, inherit.aes = TRUE)
+  Layer$new(
+    data = NULL,
+    stat = "identity",
```

Comment:
Sure, that sounds reasonable. That reminds me of another general issue which I'll add to my list above.